test/tests/materials/linear_isotropic_elasticity_tensor_soil/layered_soil_test.i

```diff
@@ -101,6 +101,7 @@
     poissons_ratio = '0.3 0.3 0.3'
     density = '2.0 2.0 2.0'
     shear_modulus = '100.0 200.0 300.0'
+    #elastic_modulus = '100.0 200.0 300.0'
```

Comment:
This line can be removed.

Reply:
But an additional test that tests the elastic modulus capability would be good.

Reply:
Sure, I added one.